### DISCUSSION TITLE
Traefik + Authelia on Unraid

### DISCUSSION THREAD

**Discussion by Starosdev** [authelia/authelia#3332] [Q&A] [answered]
### Description
So I've setup traefik, came over from NPM where I had it working with Authelia. In traefik, I can access my proxied app without any issues but as soon as I add authelia, I get a 404 error. Authelia throws this message "time="2022-05-09T22:16:08-04:00" level=info msg="Access to https://traefik.domain.com/ (method GET) is not authorized to user <anonymous>, responding with status code 302 with location redirect to https://auth.domain.com/?rd=https%3A%2F%2Ftraefik.moraleseder.com%2F&rm=GET" method=GET path=/api/verify remote_ip=172.19.0.1"

Traefik has this error message "time="2022-05-09T22:08:00-04:00" level=debug msg="Remote error http://authelia:9091/api/verify?rd=https://auth.domain.com/. StatusCode: 302" middlewareType=ForwardedAuthType middlewareName=auth@file"

### Expected Behaviour

_N/A_

### Reproduction Steps

_N/A_

### Additional Information

_N/A_

**Comment by james-d-elliott** [comment, marked as answer]
Looks fine, those logs are what are expected (the debug message is not *actually* an error, it's the expected code when a user must be authenticated). Not enough information.
